.github/workflows/pr-labels.yml

```diff
@@ -34,9 +34,10 @@ jobs:
         uses: aws-actions/configure-aws-credentials@e3dd6a429d7300a6a4c196c26e071d42e0343502 # v4.0.2
         with:
           role-to-assume: ${{ secrets.AWS_OIDC_CRIB_ROLE_ARN_SAND }}
-          role-duration-seconds: 900
-          role-session-name: gha-pr-labels-crib
           aws-region: ${{ secrets.AWS_REGION }}
+          role-duration-seconds: 900
+          mask-aws-account-id: true
+          role-session-name: pr-labels.crib
```

Comment:
this is different from before - did we change it?

Reply:
i changed it to match the format of the two existing usages of `role-session-name` - `workflow.job`, which i think is better tbh